test/functional/query-builder/update/query-builder-update.ts

```diff
@@ -150,4 +150,29 @@ describe("query builder > update", () => {
 
     })));
 
+    it("should perform update with limit correctly", () => Promise.all(connections.map(async connection => {
```

Comment:
please add checks for your test, something like this:

```ts
if (connection.driver instanceof MysqlDriver) { ... your code } else { code to handle exception since for other drivers you need to check if using those methods really throw you errors }
```